library-design-guidelines.md

```diff
@@ -95,7 +95,7 @@ they fit together, along with some more nebulous ideas of 'usability'.
    * Use `std::addressof()` to obtain addresses based on generic parameters.
    * Prefer to specify nested types as a typedef for an _unspecified_ or _implementation-defined_ type, rather than as a class or enumeration type. This avoids over-constraining implementations.
    * Do not use `requires` clauses or define new language-level
-     `concept`s until the introduction and fundamental concepts from
+     `concept`s until the introduction of fundamental concepts from
```

Comment:
Nope, these are actually two different sections. http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2017/p0802r0.html#recommendation-library-introduction vs http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2017/p0802r0.html#recommendation-fundamental-library-concepts.

Reply:
until a new introduction library introduction is written, and until the fundamental concepts from ?

Reply:
How's this?